### DISCUSSION TITLE
How to view Serenity/JS + Cucumber tests in the Testing tab of Visual Studio Code?

### DISCUSSION THREAD

**Discussion by wilmarly** [serenity-js/serenity-js#2496] [How to... ?] [answered]
If I am using the Serenity/JS+Cucumber+PlayWright template, how can I see in the Testing tab of VSC the test scenarios defined in the ".feature" files?

**Comment by jan-molak** [comment, marked as answer]
Hi @wilmarly, 

To run the tests using the Visual Studio Code UI you'll need a test runner extension, such as [CucumberJS Test Runner](https://marketplace.visualstudio.com/items?itemName=balrog994.cucumber-test-runner)
<img width="1136" alt="Screenshot 2024-08-27 at 17 01 40" src="https://github.com/user-attachments/assets/8d67b274-0f47-4742-9be9-79448cbfba7c">

For code highlighting, you'll need [Cucumber for VSCode](https://marketplace.visualstudio.com/items?itemName=CucumberOpen.cucumber-official)

**Reply by jan-molak** [reply]
Update: I've added a note on the above extensions to the [Serenity/JS Handbook](https://serenity-js.org/handbook/integration/visual-studio-code/)
